File: v4/python/image/main.py
import base64

import requests

url = 'http://api-img-bj.fengkongcloud.com/image/v4'
access_key = '{ACCESS_KEY}'
uid = '{UID}'
filename = '../files/demo.png'

with open(filename, 'rb') as f:
    content = base64.b64encode(f.read())

payload = {
    'accessKey': access_key,
    'appId': 'default',
    'eventId': 'default',
    'type': 'DEFAULT',
    'data': {
        'img': content.decode('utf-8'),
        'tokenId': uid,
    }
}

res = requests.post(url, json=payload)
# 通过 code 和 riskLevel 判断返回，具体请参考接口文档
if res:
    print(res.json())

File: v4/lua/image/main.lua
local json = require('cjson')
local mime = require("mime")
local hc = require('lib.httpclient').new()

local url = 'http://api-img-bj.fengkongcloud.com/image/v4'
local accessKey = '{ACCESS_KEY}'

local inp = assert(io.open('../files/demo.png', 'rb'))
local content = mime.b64(inp:read('*all'))

local payload = json.encode({
  accessKey = accessKey,
  appId = 'default',
  eventId = 'default',
  type = 'POLITY_QRCODE_ADVERT',
  data = {
    img = content,
    tokenId = '{UID}',
  }
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])
